lib/ts/recipe/thirdparty/types.ts

```diff
@@ -233,3 +205,5 @@ export type APIInterface = {
         | undefined
         | ((input: { code: string; state: string; options: APIOptions; userContext: any }) => Promise<void>);
 };
+
+export type TypeThirdPartyEmailDeliveryInput = TypeEmailVerificationEmailDeliveryInput;
```

Comment:
This is an incorrect type. The user object in this != the user object type for third-party recipe.

Reply:
this is true even for emailpassword recipe also right?

```
export type TypeEmailPasswordEmailDeliveryInput =
    | TypeEmailPasswordPasswordResetEmailDeliveryInput
    | TypeEmailVerificationEmailDeliveryInput;
```

Reply:
Yes. It is. So ideally should change there too.